services/rngService.js

```diff
@@ -4,11 +4,34 @@ const getRandomValue = (mean, stdev, method) => {
   return method(1, mean, stdev);
 };
 
-const getRandomInt = (min, max) => {
+const getRandomInt = (min, max, maxInclusive = false) => {
```

Comment:
Shouldn't the default be `true` to prevent creating breaking changes for current usage?